@xen-orchestra/xapi/host.mjs

```diff
@@ -31,7 +32,21 @@ class Host {
    *
    * @param {string} ref - Opaque reference of the host
    */
-  async smartReboot($defer, ref) {
+  async smartReboot($defer, ref, force = false) {
+    const residentVmRefs = await this.getField('host', ref, 'resident_VMs')
+    const vmsWithSuspendBlocked = residentVmRefs
+      .map(this.getObject)
```

Comment:
Avoid using `getObject` in `@xen-orchestra/xapi`, I'd like all methods to work there without the objects cache.